Cargo.toml

```diff
@@ -13,6 +13,8 @@ keywords = ["etcd", "v3", "api", "client", "async"]
 
 [features]
 tls = ["tonic/tls"]
+tls-openssl = ["openssl", "dep:hyper-openssl", "dep:hyper"]
```

Comment:
Currently, the actual MSRV of `etcd-client` is 1.56. The `dep:` syntax is only available starting with Rust 1.60. So do not use `dep:` here.